### DISCUSSION TITLE
Terms and Conditions

### DISCUSSION THREAD

**Discussion by mnierras** [strimzi/strimzi-kafka-operator#5101] [Q&A]
Hi!  Where can I get a copy of Strimzi's terms and conditions?  Thank you!

**Comment by scholzj** [comment]
Strimzi is an open source project licensed under Apache License version 2.0. So I guess the terms and conditions are part of the license (https://github.com/strimzi/strimzi-kafka-operator/blob/main/LICENSE)?

**Comment by mnierras** [comment]
Thank you for the clarification!

Melani

**Comment by mnierras** [comment]
Hi Jakub,

Can you also share Apache’s Data Privacy and Security Policy link?

Thank you,
Melani

**Reply by scholzj** [reply]
> Can you also share Apache’s Data Privacy and Security Policy link?

Sorry, I do not know what that is. We are not Apache Software Foundation project. We just use Apache License version 2.0 as a generally recognized open source license.

**Comment by mnierras** [comment]
Got it, thank you!

Melani
